http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -115,15 +105,14 @@ class ClusterMonitorSupervisor(
   jupyterProxyDAO: JupyterDAO,
   rstudioProxyDAO: RStudioDAO,
   welderProxyDAO: WelderDAO[IO],
-  leonardoService: LeonardoService,
   clusterHelper: ClusterHelper
 )(implicit metrics: NewRelicMetrics[IO], clusterToolToToolDao: ClusterTool => ToolDAO[ClusterTool])
     extends Actor
     with Timers
     with LazyLogging {
   import context.dispatcher
 
-  var monitoredClusters: Set[Cluster] = Set.empty
+  var monitoredClusterIds: Set[Long] = Set.empty
```

Comment:
should this be `ClusterInternalId`? since we've changed clusterId to uuid recently

Reply:
`ClusterInternalId` is a UUID, but we still have `id` which is a `Long`. Either should work here..